common/sgx_workload/workload/ext_work_order_info_kme.cpp

```diff
@@ -217,12 +220,91 @@ int ExtWorkOrderInfoKME::VerifyAttestationWpe(
 }  // ExtWorkOrderInfoKME::VerifyAttestationWpe
 
 
-int ExtWorkOrderInfoKME::CreateWorkorderKeyInfo(const ByteArray& wpe_key,
-    const ByteArray& kme_skey, ByteArray& json_key_data) {
+int ExtWorkOrderInfoKME::CreateWorkOrderKeyInfo(
+    const ByteArray& wpe_encryption_key,
```

Comment:
Add function comments.

Reply:
Its available in ext_work_order_info_kme.h. comments are huge, adding it here would make this file too big.